README.rst

```diff
@@ -35,19 +35,80 @@ In the Kinto project settings
     kinto.attachment.keep_old_files = true
 
 
+The ``gzipped`` option
+----------------------
+
 If you want uploaded files to get gzipped when stored:
 
 .. code-block:: ini
 
     kinto.attachment.gzipped = true
 
 
+The ``use_content_encoding`` option
+-----------------------------------
+
+If you want uploaded files to be Gzipped and automatically unzipped with S3 you can activate the Content-Encoding header:
+
+.. code-block:: ini
+
+    kinto.attachment.use_content_encoding = true
+
+In case you want to activate ``gzipped`` for some buckets or
+collections and activate ``use_content_encoding`` for some other, you
+can use the resource configuration key:
+
+.. code-block:: ini
+
+	# For a whole bucket (/buckets/fennec-staging)
+    kinto.attachment.resources.fennec-staging.use_content_encoding = false
+    kinto.attachment.resources.fennec-staging.gzipped = true
+
+	# Or for a specific collection in it (/buckets/fingerprinting-defenses-staging/fonts)
+    kinto.attachment.resources.fingerprinting-defenses-staging_fonts.use_content_encoding = true
+
+Be careful, ``use_content_encoding`` will force the gzipping of the files
+for the S3 storage but the metadata will look like it wasn't compressed.
```

Comment:
It's not super clear, maybe just say something like:
`Be careful, use_content_encoding automatically sets gzipped=true`